CONTRIBUTING.md

```diff
@@ -42,8 +43,9 @@ pipeline to check for common errors and possible bugs.
 ## Run this application
 
 Now that your environment is all set up, we can run the application. 
-You can get it up and running by typing `npm run dev` in the project
-root directory.
+To do that, first you need to install required dependencies. Run
+`yarn install` and once is finished, you can get it up and running
+by typing `npm run dev` in the project root directory.
```

Comment:
Since we are already using `yarn install` to install the required dependencies, what about using yarn to run the project (`yarn start`)?

Reply:
It makes sense, changing it :)